wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -127,7 +137,7 @@ public function update( $new_instance, $instance ) {
 	 * @return void
 	 */
 	public function render_media( $instance ) {
-		$instance = array_merge( $this->default_instance, $instance );
+		$instance = array_merge( wp_list_pluck( $this->get_instance_schema(), 'default' ), $instance );
```

Comment:
Why does it need to be part of the schema in the first place? Neither `rest_validate_value_from_schema()` nor `rest_sanitize_value_from_schema()` use it and it adds the overhead of having to pull it